packages/vaadin-lumo-styles/gulpfile.js

```diff
@@ -41,14 +41,15 @@ function createIconset(folder, filenames, idPrefix = '') {
     }
 
     const content = fs.readFileSync(folder + filename, 'utf-8');
-    const path = content.match(/<path d="([^"]*)"/);
+    const path = content.match(/<path( fill-rule="evenodd" clip-rule="evenodd")* d="([^"]*)"/);
     if (path) {
```

Comment:
I wonder if it would make sense to throw an exception or display a warning when nothing matches against the "path" regex. 

The current implementation silently skips the icon if `fill-rule` or `clip-rule` attributes are in a different order by an accident.

Reply:
Added an `else` clause below to throw error if an SVG file has unexpected content.